draft-annevk-johannhof-httpbis-cookies.md

```diff
@@ -651,6 +683,26 @@ cookie        = cookie-string
 cookie-string = cookie-pair *( ";" SP cookie-pair )
 ~~~
 
+While {{Section 5.4 of RFC9110}} does not define a length limit for header
+fields it is likely that the web server's implementation does impose a limit;
+many popular implementations have default limits of 8K. Servers SHOULD avoid
```

Comment:
I would prefer "8 kibibytes" here, assuming that is what this means. It's rather ambiguous in its current form.